src/test/java/com/wikia/webdriver/common/remote/operations/GetSiteId.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.common.remote.operations;
+
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.remote.operations.http.NoAuthOperation;
+import lombok.Getter;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.json.JSONObject;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class GetSiteId {
+
+  public GetSiteId(String wikiUrl) {
+    this.wikiUrl = wikiUrl;
+    this.setSiteId();
+  }
+
+  @Getter
+  private String siteId;
+  private String wikiUrl;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1213)